### ISSUE TITLE
Add rule for `Using last argument as keyword parameters`

### ISSUE THREAD

**Issue by eggplants** [rubocop/rubocop#11887]
## Is your feature request related to a problem? Please describe.

I'm newbie to Rubocop. I would like to fix some deprecated warnings to update ruby version from 2.7 to 3.0. And I tried to fix `warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call` warnings with Rubocop but couldn't fix.

## Describe the solution you'd like

```ruby
def a(a: 0, b: 0, c: 0)
  p a, b, c
end

opts = {a: 1, b: 2}

# bad
a({a: 1})
a(opts)

# good
a(a: 1)
a(**opts)
```

```ruby
def b(n, a: false, b: false)
  n.times{
    a and puts ?a
    b and puts ?b
  }
end

opts = {a: true, b: true}

# bad
b(10, {a: true, b: true})
b(10, opts)

# good
a(10, a: true, b: true)
a(10, **opts)
```

## Describe alternatives you've considered

Fix by manual.

## Additional context

...

**Comment by koic** [comment]
This warning is based on runtime information for method calls. It would be difficult to determine the validity of a combination of method definitions and calls in the RuboCop static analysis.
